### DISCUSSION TITLE
Personal page published as subdiretory

### DISCUSSION THREAD

**Discussion by jstoledano** [community/community#23214] [Other Feature Feedback, Questions, & Ideas]
<p>Hi</p>
<p>I just created a new personal subdirectory and published as personal page in master branchs, but the URL of is not formed as <code>USER.github.io</code> but <code>USER.github.io/USER</code>, like this.</p>
<p><a href="https://jstoledano.github.io/jstoledano/" class="onebox" target="_blank" rel="noopener nofollow ugc">https://jstoledano.github.io/jstoledano/</a></p>
<p>not like this</p>
<p><a href="https://jstoledano.github.io/" class="onebox" target="_blank" rel="noopener nofollow ugc">https://jstoledano.github.io/</a></p>
<p>Thanks in advance.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>It looks like your repository is named <code>jstoledano</code>. The repository for a user site must be named according to the pattern <code>&lt;username&gt;.github.io</code>, so in your case <code>jstoledano.github.io</code>. See <a href="https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/about-github-pages#types-of-github-pages-sites">Types of GitHub Pages sites</a> for details.</p>
<p>The easiest solution will probably be to rename the repository.</p>
